test/integration/targets/setup_cron/tasks/main.yml

```diff
@@ -27,6 +27,9 @@
       when: ansible_distribution != 'Alpine'
 
     - name: install faketime packages - Alpine
+      # NOTE: As of `quay.io/ansible/alpine3-test-container:4.3.0` the
```

Comment:
I wasn't sure about this but concluded that I have no idea what Alpine version this is just from the container name and thought that the comment readers wouldn't either.

Reply:
The test isn't just for our containers though -- yet the comment implies (at least to me) that the issue is somehow specific to our container, when instead it's specific to an Alpine version (and it's associated repository). I think it would be much better to simply state something like this:

> The `faketime` package is currently only available in the `edge` branch. If it ever becomes available in the `main` repository for currently tested Alpine versions, the `--repository=...` option can be dropped.

Reply:
Okay, that's fair. I'll update the wording.

Reply:
@mattclay done